health/health_log.c

```diff
@@ -191,7 +196,7 @@ static inline ssize_t health_alarm_log_read(RRDHOST *host, FILE *fp, const char
         host->health_log_entries_written++;
         line++;
 
-        int max_entries = 30, entries = 0;
+        int max_entries = 33, entries = 0; //why was it 30 ? There should have been 28 entries
```

Comment:
About your question, please, take a look on this issue https://github.com/netdata/netdata/issues/10522, it will help you to understand the motive we are using a bigger value.

Reply:
Hi Thiago! Thanks! So, you're ok with the value now being 33, right? (We add here 3 more values) ?

Reply:
Your initial thought was not completely wrong when we take a look for the majority of error logs stored, but there are few exceptions that force us to use a bigger value, 33 makes more sense here.